### PR TITLE
add External ID/Ref related descriptions

### PR DESCRIPTION
**Pull request by armintaenzertng**
This is an attempt to fill in the missing descriptions in the Core profile. Where possible, I tried to copy from the SPDX-2.3 specification.
Please note that this does not aim to be the final solution but will hopefully start discussions on possibly controversial points.

### CODE REVIEW

**Review by goneall — APPROVED** [review]
Just a cursory review, mostly of the text - but it LGTM.

I agree with the changes in the property names.

Thanks @armintaenzertng

File: model/Core/Properties/externalIdentifiers.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 ## Description
 
-TODO
+ExternalIdentifier points to a resource outside the scope of SPDX-3.0 content
+that uniquely identifies an Element.
 
 ## Metadata
 
-- name: externalIdentifiers
+- name: externalIdentifier
```
Comment (maxhbr) [inline model/Core/Properties/externalIdentifiers.md:17]: this also needs to be consistent
```suggestion
- name: externalIdentifiers
```

**Review by maxhbr — CHANGES_REQUESTED** [review]
`externalIdentifier` -> `externalIdentifiers`

**Comment by maxhbr** [comment]
same for `externalReference` -> `externalReferences`

**Review by maxhbr — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
Reviewed and approved on the 28 Mar 2023 tech call

**Pull request merged**